.github/dependabot.yml

```diff
@@ -2,10 +2,13 @@ version: 2
 updates:
   - package-ecosystem: npm
     # Files stored in repository root
-    directory: "/"
+    directory: '/'
     schedule:
-      interval: "daily"
+      interval: 'daily'
     allow:
-      - dependency-name: "@uniswap/token-lists"
-      - dependency-name: "@uniswap/default-token-list"
-      - dependency-name: "@uniswap/redux-multicall"
+      - dependency-name: '@uniswap/token-lists'
+      - dependency-name: '@uniswap/default-token-list'
+      - dependency-name: '@uniswap/redux-multicall'
+    reviewers:
+      - 'Uniswap/dependabot-reviewers'
+      - 'Uniswap/web-tokens'
```

Comment:
if there is a dependabot-reviewers team, does this need to _also_ go to web-tokens?